main/relational_to_compat.es6.js

```diff
@@ -0,0 +1,144 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const process = require('process');
+const url = require('url');
+
+require('foam2');
+
+require('../lib/property.es6.js');
+
+require('../lib/grid_dao.es6.js');
+require('../lib/http_json_dao.es6.js');
+require('../lib/json_dao_container.es6.js');
+require('../lib/local_json_dao.es6.js');
+require('../lib/web_apis/api_compat_data.es6.js');
+require('../lib/web_apis/relational_to_compat.es6.js');
+require('../lib/web_apis/release.es6.js');
+require('../lib/web_apis/release_interface_relationship.es6.js');
+require('../lib/web_apis/web_interface.es6.js');
+const pkg = org.chromium.apis.web;
+
+const logger = foam.log.ConsoleLogger.create();
+
+const USAGE = `Transforms (Release, API) pair relations to instances of
+org.chromium.apis.web.generated.CompatData. Input and output files are
+[class package path].json in [repository root]/data, with class model files in
+class:[class package path].json.
+
+USAGE:
+
+    node /path/to/relationship_to_compat.es6.js BaseConfluenceDataURL
+
+        BaseConfluenceDataURL = absolute https: or file: URL to directory
+                                where Confluence data are stored with NO
+                                trailing slash.`;
```

Comment:
Should the tool verify that there's no trailing slash on the input? Alternatively, the reads will (probably) fail quickly if you've provided a bad path, so maybe this note isn't important?

Reply:
Maybe there should be verification. My thinking was that if reads fail with a trailing slash, but you fail to notice the `//` showing up in logging of URLs, at least the usage string will provide a hint as to why it's failing.